dangerfile.js

```diff
@@ -1,31 +1,38 @@
-// eslint-disable-next-line import/no-extraneous-dependencies, no-unused-vars, object-curly-newline
-import { danger, warn, fail, message } from 'danger';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { danger, warn, fail } from 'danger';
 
-const newChangelog = danger.git.created_files.filter((filePath) => {
-  const srcFilePattern = /^packages\/([a-z-])*\/CHANGELOG.md/i;
-  return srcFilePattern.test(filePath);
-});
+const BIG_PR_THRESHOLD = 1000;
+const CHANGELOG_PATTERN = /^packages\/terra-([a-z-])*\/CHANGELOG\.md/i;
+const PACKAGE_SOURCE_PATTERN = /^packages\/terra-([a-z-])*\/src/i;
```

Comment:
We actually want this search pattern to be anything changes that wasn't the changelog. For example, if the package.json has new dependencies, the changelog needs to be updated for release. Currently if an entry isn't added, Ryan or Brett have to manually add an entry.

Reply:
5401977a5321608caab2a2b28b89139d368212ef